### PR TITLE
HT-2244: add support for serving remediate items

### PR DESCRIPTION
**Pull request by respinos**
Requires https://github.com/hathitrust/mdp-lib/pull/11.

Given an `id` and a `remediated_item_id`:

* if the package only describes one remediated file, serve that
* if the package describes multiple files, serve the package

### CODE REVIEW

**Review by aelkiss — COMMENTED** [review]
Checking how this will work -- is the expectation that the call to `/remediated` is with the original item id (e.g. hvd.32044086819505) or with the remediated item id (e.g. emma.u5f94f62cq42)?

Regardless, note that:

1) There may be multiple rows in the emma_items table with the same original_item_id, i.e. multiple remediated items for an original HT item (with different formats, coverage, remediation aspects, etc)

2) Items in the emma namespace / with the emma profile will not have entries in the rights database. Only ssd proxy users or ht_total_users should be able to access the remediated item, and then only if they can access the original item.

**Comment by respinos** [comment]
It gets invoked as: `/imgsrv/download?id=$htid;remediated_item_id=$remediated_item_id`. 
That's a different access requirement than I was given; looking into that.

**Comment by aelkiss** [comment]
Sorry, hold off for now on that -- I'm probably confused.

File: lib/SRV/Volume/Remediated/Bundle.pm [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+    unless ( defined $self->restricted ) {
+
+        my $C = $$env{'psgix.context'};
+        my $mdpItem = $C->get_object('MdpItem');
+        my $ar = $C->get_object('Access::Rights');
+        my $gId = $mdpItem->GetId();
+
+        my $final_access_status = $ar->assert_final_access_status($C, $gId);
+        my $download_access_status = $ar->get_remediated_items_access_status($C, $gId);
```
Comment (respinos) [inline lib/SRV/Volume/Remediated/Bundle.pm:195]: @aelkiss this now uses a new method in `Access::Rights` that answers the remediated files access.

**Pull request merged**